MixItUp.WPF/Controls/Chat/ChatMessageHeaderControl.xaml.cs

```diff
@@ -37,7 +37,9 @@ private void ChatMessageHeaderControl_Loaded(object sender, RoutedEventArgs e)
 
                 if (!string.IsNullOrEmpty(this.Message.User.AvatarLink))
                 {
+#pragma warning disable CS4014 // Because this call is not awaited, execution of the current method continues before the call is completed
```

Comment:
Any reason we don't want to await this call here and in other locations?

Reply:
Well it WAS async before. That is, it would download in the background. No reason to stop in this case, it'll download in the background.  Otherwise, loading like 20 avatars might be.. slow? Thoughts?

Reply:
Fair point, let's see how this goes and we can always adjust it.